### PR TITLE
Adopt POM Code Convention

### PR DESCRIPTION
**Pull request by basil**
Adopts the [POM Code Convention](https://maven.apache.org/developers/conventions/code.html#pom-code-convention) that was [voted on by Maven developers in 2008](https://lists.apache.org/thread/h8px5t6f3q59cnkzpj1t02wsoynr3ygk). Note that this PR adds no automation; this is just a one-time pass.

### CODE REVIEW

**Comment by basil** [comment]
Any takers?

**Review by jeffret-b — APPROVED** [review]

**Comment by basil** [comment]
Thanks, Jeff!

This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
